fenix/app/src/main/java/org/mozilla/fenix/shopping/ui/ReviewQualityCheckCards.kt

```diff
@@ -0,0 +1,150 @@
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Card
+import androidx.compose.material.Icon
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import org.mozilla.fenix.R
+import org.mozilla.fenix.compose.annotation.LightDarkPreview
+import org.mozilla.fenix.theme.FirefoxTheme
+
+private val cardShape = RoundedCornerShape(8.dp)
+
+/**
+ * A card with a rounded corner shape for review quality check feature that can be
+ * expanded and collapsed.
+ *
+ * @param title The title of the card.
+ * @param modifier Modifier to be applied to the card.
+ * @param content The content of the card.
+ */
+@Composable
+fun ReviewQualityCheckExpandableCard(
+    title: String,
+    modifier: Modifier = Modifier,
+    content: @Composable () -> Unit,
+) {
+    ReviewQualityCheckCard(modifier = modifier) {
+        var isExpanded by remember { mutableStateOf(false) }
+
+        Row(
+            horizontalArrangement = Arrangement.SpaceBetween,
+            modifier = Modifier
+                .fillMaxWidth()
+                .clickable {
+                    isExpanded = isExpanded.not()
+                },
+            verticalAlignment = Alignment.CenterVertically,
+        ) {
+            Text(
+                text = title,
+                color = FirefoxTheme.colors.textPrimary,
+                style = FirefoxTheme.typography.headline8,
+            )
+
+            val chevronDrawable = if (isExpanded) {
+                R.drawable.ic_chevron_up
+            } else {
+                R.drawable.ic_chevron_down
+            }
+
+            Icon(
+                painter = painterResource(id = chevronDrawable),
+                contentDescription = null,
+                tint = FirefoxTheme.colors.iconPrimary,
+                modifier = Modifier.clickable {
+                    isExpanded = isExpanded.not()
+                },
+            )
+        }
+
+        AnimatedVisibility(visible = isExpanded) {
+            content()
+        }
+    }
+}
+
+/**
+ * A card with a rounded corner shape for review quality check feature.
+ *
+ * @param modifier Modifier to be applied to the card.
+ * @param backgroundColor The background color of the card.
+ * @param elevation The elevation of the card.
+ * @param content The content of the card.
+ */
+@Composable
+fun ReviewQualityCheckCard(
+    modifier: Modifier,
+    backgroundColor: Color = FirefoxTheme.colors.layer2,
+    elevation: Dp = 1.dp,
```

Comment:
Good catch! updated! I see the elevation only in figma dev mode, is that also the only place you can see it? The [error and updated analysis cards](https://www.figma.com/file/VE5xayz3yOSB5qOATUU9Im/Mobile-Shopping-Experience-MVP?node-id=2594%3A72061&mode=dev) will be with a zero elevation and a different background color.

Reply:
I think the Figma UI changed recently, so that's the only place I see it now (though maybe someone like Crystal could point us to a different location.